openeogeotrellis/job_registry.py

```diff
@@ -929,3 +930,21 @@ def get_all_jobs_before(
             user_limit=user_limit,
         )
         return jobs
+
+    def get_running_jobs(self) -> Iterator[Dict]:
+        if self.zk_job_registry:
+            yield from self.zk_job_registry.get_running_jobs(parse_specification=True)
+        if self.elastic_job_registry:
+            return iter(self.elastic_job_registry.list_active_jobs())
+
+        return iter(())
+
+    def set_results_metadata(self, job_id, user_id, costs: Optional[float], usage: dict,
+                             results_metadata: Dict[str, Any]):
+        if self.zk_job_registry:
+            self.zk_job_registry.patch(job_id=job_id, user_id=user_id,
+                                       **dict(results_metadata, costs=costs, usage=usage))
+
+        if self.elastic_job_registry:
+            self.elastic_job_registry.set_results_metadata(job_id=job_id, costs=costs, usage=usage,
+                                                           results_metadata=results_metadata)
```

Comment:
Write a test for this method.

Reply:
Covered with `tests.test_job_registry.TestDoubleJobRegistry.test_set_results_metadata` and `tests.test_job_registry.TestDoubleJobRegistry.test_get_results_metadata`.